jib-core/src/main/java/com/google/cloud/tools/jib/image/LayerEntry.java

```diff
@@ -87,12 +104,12 @@ public Path getExtractionPath() {
   }
 
   /**
-   * Gets the source file path in string form.
+   * Gets the source file absolute path in string form.
```

Comment:
How about "Get the source file as an absolute path in string form. The path is made to absolute first, if not already absolute."

And do this for all the other `get...FileString()` methods.

Also, I think it's worth adding a test that a relative path is actually converted to an absolute path.

Reply:
Maybe renaming `getSourceFileString` to `getAbsoluteSourceFileString` as well?